website/content/docs/v0.7/Local Platforms/virtualbox.md

```diff
@@ -0,0 +1,166 @@
+You can also repeat this for additional nodes desired.
+
+## Start Control Plane Node
+
+Once the VMs have been created and updated, start the VM that will be the first control plane node.
+This VM will boot the ISO image specified earlier and enter "maintenance mode".
+Once the machine has entered maintenance mode, there will be a console log that details the IP address that the node received.
+Take note of this IP address, which will be referred to as `$CONTROL_PLANE_IP` for the rest of this guide.
+
+## Generate Machine Configurations
+
+With the IP address above, you can now generate the machine configurations to use for installing Talos and Kubernetes.
+Issue the following command, updating the output directory, cluster name, and control plane IP as you see fit:
+
+```bash
+talosctl gen config talos-vbox-cluster https://$CONTROL_PLANE_IP --output-dir _out
+```
+
+This will create several files in the _out directory: init.yaml, controlplane.yaml, join.yaml, and talosconfig.
+
+## Create Control Plane Node
+
+Using the `init.yaml` generated above, you can now apply this config using talosctl.
+Issue:
+
+```bash
+talosctl apply-config --insecure --nodes $CONTROL_PLANE_IP --file _out/init.yaml
+```
+
+You should now see some action in the Virtualbox console for this VM.
+Talos will be installed to disk, the VM will reboot, and then Talos will configure the Kubernetes control plane on this VM.
+
+> Note: This process can be repeated multiple times to create an HA control plane.
+> Simply apply `controlplane.yaml` instead of `init.yaml` for subsequent nodes.
+
+## Create Worker Node
+
+Create at least a single worker node using a process similar to the control plane creation above.
+Start the worker node VM and wait for it to enter "maintenance mode".
+Take note of the worker node's IP address, which will be referred to as `$WORKER_IP`
+
+Issue:
+
+```bash
+talosctl apply-config --insecure --nodes $WORKER_IP --file _out/join.yaml
+```
+
+> Note: This process can be repeated multiple times to add additional workers.
+
+## Using the Cluster
+
+Once the cluster is available, you can make use of `talosctl` and `kubectl` to interact with the cluster.
+For example, to view current running containers, run `talosctl containers` for a list of containers in the `system` namespace, or `talosctl containers -k` for the `k8s.io` namespace.
+To view the logs of a container, use `talosctl logs <container>` or `talosctl logs -k <container>`.
+
+First, configure talosctl to talk to your control plane node by issuing the following, updating paths and IPs as necessary:
+
+```bash
+export TALOSCONFIG="_out/talosconfig"
+talosctl config endpoint $CONTROL_PLANE_IP
+talosctl config node $CONTROL_PLANE_IP
+```
+
+## Retrieve and Configure the `kubeconfig`
+
+Fetch the kubeconfig file from the control plane node by issuing:
+
+```bash
+talosctl kubeconfig .
+```
+
+You can then use kubectl in this fashion:
+
+```bash
+kubectl --kubeconfig kubeconfig get nodes
```

Comment:
this should be without `--kubeconfig kubeconfig`, as by default `talosctl kubeconfig` is going to append the context to the `~/.kube/config`

Reply:
good call. fixed.